src/Components/AddNewAds/AddNewAds.jsx

```diff
@@ -0,0 +1,200 @@
+import {
+  Box,
+  Button,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  MenuItem,
+  Stack,
+  TextField,
+  Typography,
+} from "@mui/material";
+import axios from "axios";
+import React, { useContext, useEffect, useState } from "react";
+import { useForm } from "react-hook-form";
+import { AuthContext } from "../Context/AuthContext/AuthContext";
+import { toast } from "react-toastify";
+import { Link, useNavigate } from "react-router-dom";
+import KeyboardDoubleArrowLeftIcon from "@mui/icons-material/KeyboardDoubleArrowLeft";
+
+const currencies = [
+  {
+    value: "true",
+    label: "true",
+  },
+  {
+    value: "false",
+    label: "false",
+  },
+];
+export default function AddNewAds() {
+  const navigate = useNavigate();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({ defaultValues: { isActive: "true" } });
+  const handleClose = () => {
+    setOpen(false);
+    setModelState(close);
+  };
+
+  const [roomsList, setRoomsList] = useState([]);
+  const [isLoding, setIsLoding] = useState(false);
+  const { baseUrl, requstHeaders, userRole } = useContext(AuthContext);
+  // ****************** to add new Ads ***************************************************************************************
+  const onSubmit = (data) => {
+    setIsLoding(true);
+
+    axios
+      .post(`${baseUrl}/admin/ads`, data, {
+        headers: requstHeaders,
+      })
+      .then((response) => {
+        toast.success(response?.data?.message || "Added Successfully");
+        navigate("/dashboard/ads");
+      })
+      .catch((error) => {
+        toast.error(error?.response?.data?.message || "Data of task invaild");
```

Comment:
You can change the message here to `error validating your request`, as there is no task in this form.